content/en-us/tutorials/fundamentals/coding-1/creating-a-script.md

```diff
@@ -159,14 +159,14 @@ Print functions display text on the screen, as you saw earlier. It's one of the
 
 ### Combining Strings
 
-You can display any string in the Output using `print()`; you can even print multiple strings stored within variables or typed directly within the function. **Concatenation** is combining strings. To concatenate the string assigned to your variable and a second string, use two dots `..` The following example concatenates two variables and two typed strings.
+You can display any string in the Output using `print()`; you can even print multiple strings stored within variables or typed directly within the function. **Interpolation** is combining strings. To interpolate the string assigned to your variable and a second string, use backticks and brackets (`{}`). The following example interpolates two variables and two typed strings.
```

Comment:
Consider removing `()` so that it doesn't confuse the syntax

Reply:
Also these backticks will probably be interpreted by the markdown renderer and now show up in the actual syntax example. Probably need to do something like
``` `` `{}` `` ```
to get it to show up correctly

Reply:
I have no idea if it’s even possible for to have backticks as code lol, will try it out though.

Reply:
* https://stackoverflow.com/questions/55586867/how-to-put-in-markdown-an-inline-code-block-that-only-contains-a-backtick-char
* https://daringfireball.net/projects/markdown/syntax#code

![image](https://github.com/Roblox/creator-docs/assets/66378309/717fd2eb-e29e-41c8-a390-04bb6df3a726)

Reply:
> Consider removing `()` so that it doesn't confuse the syntax

Where should I remove the parenthesis? In the `print()` or (`` `{}` ``)?
(Also, made the backticks show up together with the brackets)

Reply:
Was referring to the (`` `{}` ``)